packages/html/__tests__/HtmlVideoLayer.test.ts

```diff
@@ -0,0 +1,16 @@
+import {CloudinaryVideo} from "@cloudinary/url-gen";
+import {HtmlVideoLayer} from "../src";
+
+describe('HtmlVideoLayer tests', function () {
+    const cldVideo = new CloudinaryVideo('sample', {cloudName: 'demo'});
+
+    it('should set video src', function () {
+        const video = document.createElement('video');
+        new HtmlVideoLayer(video, cldVideo, null);
+        process.nextTick(() => {
+            expect((video.children[0] as HTMLSourceElement).src).toEqual("https://res.cloudinary.com/demo/video/upload/sample.webm?_a=ATRSRAA0");
```

Comment:
It looks like analytics contains the actual token, which will break on the next version. I fix the text and search for the actual indicator( like check that it includes file extension or something unique to that item.

Reply:
@const-cloudinary i removed the `HtmlVideoLayer` tests since i found a bug in. This layer does not use the correct analytics codes so i will fix it and then bring back those tests